backend/cinema_service/settings.py

```diff
@@ -28,7 +28,7 @@
 # SECURITY WARNING: don't run with debug turned on in production!
 DEBUG = True
```

Comment:
In a production environment, you should set DEBUG to False to avoid exposing sensitive information. Consider using environment variables to manage this setting depending on the environment.